homeassistant/components/sunsynkweb/__init__.py

```diff
@@ -0,0 +1,28 @@
+"""The Sunsynk Inverter Web integration."""
+
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import PlantUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
```suggestion

type SunsyncConfigEntry = ConfigEntry[PlantUpdateCoordinator]

async def async_setup_entry(hass: HomeAssistant, entry: SunsyncConfigEntry ) -> bool:
```

to type the `runtime_data` correctly, also replace the `ConfigEntry` with `SunsyncConfigEntry ` in `coordinator.py`